src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -228,6 +228,68 @@ public string[] PreContent
         private string[] _preContent;
 
         /// <summary>
+        /// Sets and Gets the meta property of the HTML head
+        /// </summary>
+        /// <returns></returns>
+        [Parameter(ParameterSetName = "Page")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable Meta
+        {
+            get
+            {
+                return _meta;
+            }
+            set
+            {
+                _meta = value;
+                _metaSpecified = true;
+            }
+        }
+        private Hashtable _meta;
+        private bool _metaSpecified = false;
+
+        /// <summary>
+        /// Specifies the charset encoding for the HTML document
+        /// </summary>
+        [Parameter(ParameterSetName = "Page")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("UTF-8", "ISO-8859-1","ASCII","ANSI",IgnoreCase=true)]
```

Comment:
Technically, the allowed set of (legacy) encodings is quite large although HTML5 I believe has standardized on UTF-8.  So perhaps this shouldn't have a `ValidateSet`?